android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -714,6 +730,13 @@ constructor(
           searchPath = "${resourceGroup.key}/${it.focus.extractId()}",
           patientRelatedResourceTypes = patientRelatedResourceTypes,
         )
+        if (resourceGroup.key.contains("List")) {
+          val searchPath = "${resourceGroup.key}/${it.focus.extractId()}"
+          val resourceId = searchPath.substringAfter(TYPE_REFERENCE_DELIMITER).trim()
+          val listResourceUrlPath =
+            "${ResourceType.List.name}?$ID=$resourceId&_page=${applicationConfig.listSyncConfig.page}&_count=${applicationConfig.listSyncConfig.count}"
+          fhirResourceDataSource.getResource(listResourceUrlPath)
+        }
```

Comment:
I don't think we need this check and code block here. The parent method `processCompositionListResources` solely processes List resource type

Reply:
Ooh okay. Changing that as well